### ISSUE TITLE
The raw statistics are in ns, not us (documentation needs update)

### ISSUE THREAD

**Issue by prutseltje** [koct9i/ioping#38]
The raw statistics are printed in ns, and not in us. At least since some 0.9 versions.

I have tested this on 2 Debian versions and with the latest git source:

Debian jessie (8): (correct output in usec)
```
$ ioping -v
ioping 0.9

$ ioping -i 1 -p 5 -W /tmp/
4 KiB to /tmp/ (ext4 /dev/mapper/vg0-tmp): request=1 time=151 us
4 KiB to /tmp/ (ext4 /dev/mapper/vg0-tmp): request=2 time=141 us
4 KiB to /tmp/ (ext4 /dev/mapper/vg0-tmp): request=3 time=138 us
4 KiB to /tmp/ (ext4 /dev/mapper/vg0-tmp): request=4 time=143 us
4 KiB to /tmp/ (ext4 /dev/mapper/vg0-tmp): request=5 time=175 us
5 748 6684 27379679 138 150 175 13
^C
--- /tmp/ (ext4 /dev/mapper/vg0-tmp) ioping statistics ---
5 requests completed in 4.86 s, 6.68 k iops, 26.1 MiB/s
min/avg/max/mdev = 138 us / 149 us / 175 us / 13 us
```

Debian stretch (9): (incorrect output, manpage says usec for raw output)
```
$ ioping -v
ioping 0.9

$ ioping -i 1 -p 5 -W /tmp/
4 KiB >>> /tmp/ (ext4 /dev/sda2): request=1 time=4.89 ms (warmup)
4 KiB >>> /tmp/ (ext4 /dev/sda2): request=2 time=2.31 ms
4 KiB >>> /tmp/ (ext4 /dev/sda2): request=3 time=9.09 ms
4 KiB >>> /tmp/ (ext4 /dev/sda2): request=4 time=1.97 ms
4 KiB >>> /tmp/ (ext4 /dev/sda2): request=5 time=2.67 ms
4 16033870 249 1021837 1972563 4008468 9088666 2943301 5 4002766704
^C
--- /tmp/ (ext4 /dev/sda2) ioping statistics ---
4 requests completed in 16.0 ms, 16 KiB written, 249 iops, 997.9 KiB/s
generated 5 requests in 4.44 s, 20 KiB, 1 iops, 4.51 KiB/s
min/avg/max/mdev = 1.97 ms / 4.01 ms / 9.09 ms / 2.94 ms
```

Latest git: (output in ns, but documentation says usec)
```
$ ./ioping -v
ioping 1.0.12.g5bd3232

$ ./ioping -i 1 -p 5 -W /tmp/
4 KiB >>> /tmp/ (ext4 /dev/dm-1): request=1 time=163.9 us (warmup)
4 KiB >>> /tmp/ (ext4 /dev/dm-1): request=2 time=135.3 us
4 KiB >>> /tmp/ (ext4 /dev/dm-1): request=3 time=106.2 us
4 KiB >>> /tmp/ (ext4 /dev/dm-1): request=4 time=111.7 us
4 KiB >>> /tmp/ (ext4 /dev/dm-1): request=5 time=128.1 us
4 481326 8310 34039300 106209 120332 135300 11808 5 4000234525
^C
--- /tmp/ (ext4 /dev/dm-1) ioping statistics ---
4 requests completed in 481.3 us, 16 KiB written, 8.31 k iops, 32.5 MiB/s
generated 5 requests in 4.51 s, 20 KiB, 1 iops, 4.44 KiB/s
min/avg/max/mdev = 106.2 us / 120.3 us / 135.3 us / 11.8 us
```

**Comment by jimis** [comment]
So I am using a custom script with `ioping -B` to measure I/O latency, and I notice that the numbers written to the database are in different units, depending on the OS. I thought the "raw" format was intended to be parsed by machines, and thus stay stable across versions. Why did the units change?

**Comment by koct9i** [comment]
Yep, all should be in nanoseconds. I've fixed manpage in 1.1
